src/fn/fn-select.scss

```diff
@@ -57,13 +57,20 @@ $block: #{$fn-namespace}-select;
     padding: $fn-padding-xs $fn-padding-small;
 
     &::before {
-      @include fn-set-position-left(0);
-
       top: 0;
+      left: 0;
       content: '';
       height: 100%;
-      width: 0.125rem;
       position: absolute;
+      border-left-width: 2px;
+      border-left-style: solid;
+    }
+
+    @include fn-rtl() {
```

Comment:
Previously used mixin was doing the same:

```
@mixin fn-set-position-left($left) {
  left: $left;

  @include fn-rtl() {
    left: auto;
    right: $left;
  }
}
```

Reply:
it does not work. for before/after